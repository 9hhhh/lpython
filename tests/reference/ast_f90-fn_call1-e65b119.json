{
    "basename": "ast_f90-fn_call1-e65b119",
    "cmd": "lfortran --show-ast-f90 --no-color {infile}",
    "infile": "tests/fn_call1.f90",
    "infile_hash": "734b71292f6dc8ea73b211bbddc7438c34a8a3bd72ccbffa35f19e0e",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast_f90-fn_call1-e65b119.stdout",
    "stdout_hash": "57464594a9c140984450d6a8f2864ab9e061bd798be860666065610b",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}