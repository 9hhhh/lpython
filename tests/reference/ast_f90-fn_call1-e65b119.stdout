size(tvec, 2)
min(tvec(2, i), tvec(1, i))
a(1, 2, 3)
A(i, j, k)